scss/tech.scss

```diff
@@ -0,0 +1,42 @@
+.tech {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-flow: column wrap;
+  margin: 15px;
+  padding: 10px;
+  position: relative;
+  text-align: center;
+  width: 190px;
+
+  &::before {
+    border-radius: 5px;
+    border: 1px solid $green;
+    content: '';
+    height: 241px;
+    margin: 10px;
+    position: absolute;
+    right: 0;
+    top: -20px;
+    width: 189px;
+    z-index: -1;
+  }
+}
+
+.tech-img {
+  width: 100px;
+}
+
+.tech-title,
+.tech-release {
+  padding: 5px 0;
+}
+
+.tech-release {
+  font-weight: bold;
+}
+
+.tech-desc {
+  padding: 15px 0;
```

Comment:
you can remove this if you add margin bottom of 15px to the tech-release and the other 15 are not needed here